ganga/GangaCore/Lib/Localhost/Localhost.py

```diff
@@ -184,10 +186,16 @@ def preparejob(self, jobconfig, master_input_sandbox):
         import inspect
 
         fileutils = File( inspect.getsourcefile(GangaCore.Utility.files), subdir=PYTHON_DIR )
+        virtualizationutils = File( inspect.getsourcefile(GangaCore.Utility.Virtualization), subdir=PYTHON_DIR )
 
         sharedfiles = jobconfig.getSharedFiles()
 
-        subjob_input_sandbox = job.createPackedInputSandbox(jobconfig.getSandboxFiles() + [ fileutils ] )
+        subjob_input_sandbox = job.createPackedInputSandbox(jobconfig.getSandboxFiles() + [ fileutils, virtualizationutils] )
+        
+        import tempfile
+        workdir = tempfile.mkdtemp(dir=config['location'])
```

Comment:
Why did these two lines move? They are not directly used by your changes right?

Reply:
fixed